TileStache/Goodies/VecTiles/sort.py

```diff
@@ -32,7 +32,7 @@ def _by_scalerank(feature):
 def _by_population(feature):
     wkb, properties, fid = feature
     value_for_none = -1000
-    population = properties.get('population', value_for_none)
+    population = int(properties.get('population', value_for_none))
```

Comment:
Yup, good idea. People put all sorts in there, including thousands separators. Probably would be good to be more robust for all kinds of numeric parsing, but that's something for another day...